build.sbt

```diff
@@ -49,20 +49,13 @@ lazy val commonSettings = Seq(
   javaOptions in (Test, run) ++= Seq("-Xms64m", "-Xmx64m"),
   libraryDependencies ++= Seq(
     compilerPlugin("org.typelevel" %% "kind-projector" % "0.10.0"),
-    "org.typelevel" %%% "cats-core" % "2.0.0-M1",
-    "org.typelevel" %%% "cats-laws" % "2.0.0-M1" % "test",
-    "org.typelevel" %%% "cats-effect" % "2.0.0-M1",
-    "org.typelevel" %%% "cats-effect-laws" % "2.0.0-M1" % "test",
-    "org.scalatest" %%% "scalatest" % "3.0.8-RC2" % "test"
+    "org.typelevel" %%% "cats-core" % "2.0.0-M2",
+    "org.typelevel" %%% "cats-laws" % "2.0.0-M2" % "test",
+    "org.typelevel" %%% "cats-effect" % "2.0.0-M2",
+    "org.typelevel" %%% "cats-effect-laws" % "2.0.0-M2" % "test",
+    "org.scalacheck" %%% "scalacheck" % "1.14.0" % "test",
```

Comment:
If we're on cats-laws-2.x and cats-effect-laws-2.x, we should be on scalacheck-1.14.0.